6-AdvancedScenarios/1-call-api-obo/README.md

```diff
@@ -1,427 +1,361 @@
-        - For **User consent display name** type `Access msal-react-downstream`.
-        - For **User consent description** type `Allow the application to access msal-react-downstream on your behalf.`
-        - Keep **State** as **Enabled**.
-        - Select the **Add scope** button on the bottom to save this scope.
-1. On the right side menu, select the `Manifest` blade.
-   - Set `accessTokenAcceptedVersion` property to **2**.
-   - Click on **Save**.
-
-#### Configure the downstream web API (msal-react-downstream) to use your app registration
-
-Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
-
-> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
-
-1. Open the `DownstreamAPI\config.json` file.
-1. Find the key `clientID` and replace the existing value with the application ID (clientId) of `msal-react-downstream` app copied from the Azure portal.
-1. Find the key `tenantID` and replace the existing value with your Azure AD tenant ID.
-
-### Register the middle-tier web API (msal-react-middletier)
+#### Register the service app (msal-react-api)
 
-1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure AD** service.
+1. Navigate to the [Azure portal](https://portal.azure.com) and select the **Azure Active Directory** service.
 1. Select the **App Registrations** blade on the left, then select **New registration**.
 1. In the **Register an application page** that appears, enter your application's registration information:
-   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-react-middletier`.
-   - Under **Supported account types**, select **Accounts in this organizational directory only**.
-1. Select **Register** to create the application.
-1. In the app's registration screen, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
-1. Select **Save** to save your changes.
-1. In the app's registration screen, select the **Certificates & secrets** blade in the left to open the page where we can generate secrets and upload certificates.
+    1. In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `msal-react-api`.
+    1. Under **Supported account types**, select **Accounts in this organizational directory only**
+    1. Select **Register** to create the application.
+1. In the **Overview** blade, find and note the **Application (client) ID**. You use this value in your app's configuration file(s) later in your code.
+1. In the app's registration screen, select the **Authentication** blade to the left.
+1. If you don't have a platform added, select **Add a platform** and select the **Web** option.
+    1. In the **Redirect URI** section enter the following redirect URI:
+        1. ``
```

Comment:
Redirect URI missing (possibly a codegen issue)

Reply:
+1. coordinate with @v-michaelmi

Reply:
No issue here. I reran the codegen and it works as expected.